addons/hr_recruitment/models/hr_job.py

```diff
@@ -108,7 +108,15 @@ def _alias_get_creation_values(self):
     @api.model
     def create(self, vals):
         vals['favorite_user_ids'] = vals.get('favorite_user_ids', []) + [(4, self.env.uid)]
-        return super(Job, self).create(vals)
+        new_job = super(Job, self).create(vals)
+        utm_linkedin = self.env['utm.source'].search([["name", "ilike", "linkedin"]])
```

Comment:
You have to give a tuple list for the ORM function: `utm_linkedin = self.env['utm.source'].search([('name', 'ilike', 'linkedin')])`
The same for the line below